solvers/+drakeFunction/+geometry/Cartesian2CylindricalTransform.m

```diff
@@ -0,0 +1,49 @@
+classdef Cartesian2CylindricalTransform < drakeFunction.DrakeFunction
+  % Given a cartesian coordinate frame, transform its coordinates to a
+  % cylindrical coordinates
+  properties
+    T_cylinder % This is a homogeneous transformation matrix, T_cylinder transforms a cylinder with [0;0;1] being its axis, and [1;0;0] as its theta=0 radius, to the cylinder in the cartesian frame
```

Comment:
The part 'and [1;0;0] as its theta=0 radius' is a little hard to understand. Maybe change it to 'and the angle theta measured with respect to the x-axis'.